### PR TITLE
fix workspace

### PR DESCRIPTION
**Pull request by namframgia**

### CODE REVIEW

**Comment by namframgia** [comment]
https://edu-redmine.framgia.vn/issues/5117

File: app/Http/Controllers/Admin/WorkspaceController.php [hunk 1]
```diff
@@ -25,9 +26,21 @@ public function __construct(WorkspaceRepository $workspaceRepository)
      */
     public function index()
     {
-        $workspaces = $this->workspace->getWorkspaces();
+        $workspaces = $this->workspace->with(['locations' => function ($query) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/WorkspaceController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/WorkspaceController.php</br>**Line**: 29</br>Each line in an array declaration must end in a comma

File: app/Http/Controllers/Admin/WorkspaceController.php [hunk 2]
```diff
@@ -25,9 +26,21 @@ public function __construct(WorkspaceRepository $workspaceRepository)
...
+                ->select(DB::raw('location_id, count(*) as total_seat'))
+                ->where(DB::raw('seats.usable'), '!=', config('site.default.unusable'))
+                ->orWhere('seats.usable')
+                ->groupBy('location_id');
 
-        return view('admin.workspace.list', compact('workspaces'));
+            $query->joinSub($seats, 'Total', function ($join) {
+                $join->on('locations.id', 'Total.location_id');
+            });
+        }])->get();
```
Comment (framgiaciv3) [inline app/Http/Controllers/Admin/WorkspaceController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.CloseBraceNewLine</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/WorkspaceController.php</br>**Line**: 39</br>Closing bracket of array declaration must be on a new line

File: resources/views/test/workspace/index.blade.php [hunk 3]
```diff
@@ -63,30 +68,49 @@
...
+                                            <a class="edit-workspace btn btn-warning m-btn m-btn--icon m-btn--icon-only m-btn--custom m-btn--pill" 
+                                                data-toggle="modal" 
+                                                data-target="#m_modal_4"
+                                                onclick="showModal({{ $item->id }}, '{{ $item->name }}', '{{ $item->photo }}')"
+                                            >
                                                 <i class="flaticon-edit-1"></i>
                                             </a>
                                         @endif
-
+
```
Comment (BunnyPi04) [inline resources/views/test/workspace/index.blade.php:113]: dư space
xóa vào pull sau nhé

**Pull request merged**